stitching/stitcher.py

```diff
@@ -84,10 +88,15 @@ def initialize_stitcher(self, **kwargs):
         self.blender = Blender(args.blender_type, args.blend_strength)
         self.timelapser = Timelapser(args.timelapse)
 
-    def stitch(self, img_names):
-        self.initialize_registration(img_names)
+    def stitch(self, img_names, mask_names=None):
+        self.initialize_registration(img_names, mask_names)
         imgs = self.resize_medium_resolution()
-        features = self.find_features(imgs)
+        if mask_names is None :
+            feature_masks = None
+        else:
+            feature_masks = list(self.mask_handler.to_medium_resolution_binary_mask())
```

Comment:
without `to_medium_resolution_binary_mask` we would first resize the masks to medium resolution and then convert them to binary or vice versa

Reply:
Agree, i fee it is related to the new ImageHandler changes am i right ?

Reply:
yes